### DISCUSSION TITLE
Raspberry 3B

### DISCUSSION THREAD

**Discussion by agustinvaquero** [onlaj/Piano-LED-Visualizer#416] [General]
It is possible run the project into Raspberry 3B ?
I Follow all the steps but I dont see anything... only obtain an infinite loop with message: ... Listening on a socket...
Thank you.

**Comment by onlaj** [comment]
Hey, did you connect your RPi to the internet?

**Comment by agustinvaquero** [comment]
Yes By Wifi

**Comment by agustinvaquero** [comment]
But I dont Kwnow how visualizate the app ...

**Comment by agustinvaquero** [comment]
Hi... The web interface is great, and is working fine... in my case I connect to http://192.168.0.23/ and show all the options and works perfectly. The led strip responds to piano keystrokes. But i dont have access to the App interface... with the menu options... I don´t Known if is necesary or not. My target is play my midi songs, and connect it to syntesia at pc with Windows 10. Thanks !!

**Comment by onlaj** [comment]
Hey, if the web interface is working for you there is no need for a screen's menu.
